src/helper/args/mod.rs

```diff
@@ -89,52 +87,43 @@ fn check_args<'a, ArgsIter: Iterator<Item = &'a str>, Output: Write>(
 /// Shows command-line help about the given command.
 pub fn get_args_help<Output: Write>(
     cmd: &str,
-    cli_args: &CliArgs,
-    config: Option<Config>,
+    config: &Config,
+    verbose: bool,
     output: &mut Output,
 ) -> Result<()> {
-    if let Some(config_args) = config.and_then(|v| v.check_args) {
-        for args in config_args {
+    if cmd.trim().is_empty() {
+        return Ok(());
+    }
+
+    if let Some(ref args) = config.check_args {
+        if args.is_empty() {
+            return Ok(());
+        }
+        for arg_variants in [
+            (config.check_version).then(|| &args[0]),
+            (config.check_help && args.len() >= 2).then(|| &args[1]),
```

Comment:
I didn't quite understand the logic here, why use the first and second elements of `args` and how do we guarantee that they will be present?

Reply:
I already checked if the array is empty above, In an ideal situation the `args` array should have two arrays inside, the first one for the check version arguments, and this should exist if the `args` array is not empty, and the second array for the check help arguments this does maybe not exist even if the user enables the `check_help` option

Reply:
Okay I see, let's keep it like this then.